### PR TITLE
[Security] Add allowed_connections commands

### PR DESCRIPTION
**Pull request by ariklin**
**Description**  
[Security] Add allowed_connections commands

Add allowed_connections commands, list of all possible traffic between resources for the subscription and location.

**Testing Guide**  
Users can get a list, single (show) of list of all possible traffic between resources:

az security allowed_connections list
az security allowed_connections show --name Internal --resource-group mygroup

Tests are included, use:  azdev test security test_allowed_connections --live
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [*] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [*] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
add to S172

**Comment by ariklin** [comment]
@haroldrandom - fixed as requested.

**Comment by ariklin** [comment]
> As for code style:
> 
> 1. Too many local variables (26/25) (too-many-locals) —— you could add # pylint: disable=too-many-locals
>    As for command design:
> 2. You leave one parameter without help message.

@haroldrandom,  thank you for your guidance, please consider my last commit

File: .gitignore [hunk 1]
```diff
@@ -6,6 +6,7 @@ __pycache__/
 env/
 env27/
 venv/
+myEnv/
```
Comment (haroldrandom) [inline .gitignore:9]: This is not necessary. Just leave it untouch.

**Pull request merged**